src/main/java/org/folio/rest/impl/TenantReferenceAPI.java

```diff
@@ -66,6 +66,11 @@ private void buildDataLoadingParameters(TenantAttributes tenantAttributes, Tenan
         .withLead("data")
         .add("interfaces-2.0.0", "organizations-storage/interfaces");
       }
+      if (isNew(tenantAttributes, "4.7.0")) {
```

Comment:
We need to re-check in our POs do we need to add reference data for priviledge contacts

Reply:
I checked and we don't need them